policies/stable-release-updates.md

```diff
@@ -0,0 +1,91 @@
+Stable Release Update Policy
```

Comment:
PR title and filename says "Updates"  but the title says "Update". Probably "Updates" is better.